v2/emailpassword/serverless/with-aws-lambda/authorizer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: authorizer
+title: Lambda Authorizer
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/serverless/with-aws-lambda/authorizer.mdx -->
+
+# Lambda Authorizer
+
+You can use a lambda similar to [step 5](./session-verification) as an Authorizer in API Gateways. This will enable you to use SuperTokens in a lambda to authorize requests to other integrations (e.g., AppSync).
```

Comment:
Maybe you should add somewhere something like "This authorisor will do session verification and if successful, will add the userId of the logged in user to the request object as the `x-user-id` header. If no session exists, this header won't exist either.